app/controllers/jobs_controller.rb

```diff
@@ -52,11 +52,13 @@ def enabled
   end
 
   def destroy
-    if @job.can_be_stopped_by?(current_user)
+    # if @job.can_be_stopped_by?(current_user)
```

Comment:
wanted to leave it as reminder that this is broken / not working as intended ... since you should always be able to stop the jobs you created